### ISSUE TITLE
复现失败

### ISSUE THREAD

**Issue by 1996Eric** [bubbliiiing/mask-rcnn-keras#44]
博主您好，我在使用您提供的权重复现代码时候，测试street图片出现了这样的情况，
<img width="558" alt="2022-12-07" src="https://user-images.githubusercontent.com/104717289/206167769-5ed23b77-8451-4752-b04b-79034d97b362.png">

**Comment by bubbliiiing** [comment]
嗯？环境是？

**Comment by 1996Eric** [comment]
电脑前几天重新装了，环境也重新弄了，具体版本没记住应该是cuda10.0或是10.2，其他都是按照里面的要求来的，**之前在旧电脑上就可以成功复现**，新电脑上就出现了这个问题。然后我尝试复现retinanet-pytorch时也出现了同样的问题，有时候是没有识别直接输出原图，有时候就像上图一样，电脑是win11，3070ti，cuda10.0，python3.7，pytorch1.2，其他的版本也都对应。您知道是什么原因么？

**Comment by 1996Eric** [comment]
您好！我试了一下cpu版本的，就正常了...

**Comment by bubbliiiing** [comment]
鹅3070不支持cuda10
